demos/python_demos/llm_text_generation/servable_unary/config.py

```diff
@@ -71,114 +65,265 @@ def youri_partial_text_processor(partial_text, new_text):
     return partial_text
 
 
+def internlm_partial_text_processor(partial_text, new_text):
+    partial_text += new_text
+    return partial_text.split("<|im_end|>")[0]
+
+
 SUPPORTED_LLM_MODELS = {
```

Comment:
Same as for stream. Could we have a comment about entries that were commented out?